packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -4074,7 +4075,8 @@ void main() {
 
     await tester.longPressAt(textfieldStart + const Offset(50.0, 2.0));
     await tester.pump(const Duration(milliseconds: 150));
-    await tester.tapAt(textfieldStart + const Offset(20.0, 146.0));
+    // Tap the Select All button.
+    await tester.tapAt(textfieldStart + const Offset(20.0, 100.0));
```

Comment:
Why did these y coordinates need to change?

Reply:
The position of various text editor UI elements such as the selection handles is related to the caret position, and apparently the Cupertino "select all" button is affected.  I don't know exactly how that position is calculated though.

Reply:
Ah of course.  Thanks for the explanation.